src/components/BottomSheet/BottomSheetMenu.js

```diff
@@ -53,15 +53,15 @@ const styles = StyleSheet.create({
     borderBottomWidth: 1,
     paddingVertical: dimen.normalizeDimen(14),
     paddingHorizontal: dimen.normalizeDimen(20),
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
   },
   rbsheetContent2: {
     flexDirection: 'row',
     alignItems: 'center',
     borderBottomWidth: 1,
     paddingVertical: dimen.normalizeDimen(14),
     paddingHorizontal: dimen.normalizeDimen(20),
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change in border color from `COLORS.greySubtile2` to `COLORS.gray200` should be fine if it aligns with the overall design and theme of the application. Make sure this change doesn't negatively impact the visibility of the borders or the overall user interface.

```diff
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
```
```diff
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->